xml/favicons.xml

```diff
@@ -96,20 +97,22 @@
   </fingerprint>
 
   <fingerprint pattern="^4644f2d45601037b8423d45e13194c93$">
-    <description>Tomcat</description>
+    <description>Apache Tomcat (5.5.26), Alfresco Community</description>
```

Comment:
Already talking about this in a DM; but looks like some of these fingerprints might've come from nmap originally https://github.com/nmap/nmap/blob/master/nselib/data/favicon-db#L92C1-L92C33

This is likely another scenario we would have to remove the version details from